### ISSUE TITLE
Not being able to see visuals at all (Linux/)

### ISSUE THREAD

**Issue by edumeow** [danielkrupinski/Osiris#4221]
Other features work visuals dont..

**Comment by Q-666** [comment]
having the same problem, cant see the footstep visuals

**Comment by edumeow** [comment]
i figured it out dont use snap or flatpak steam use native install of steam website for the deb file

**Comment by prontodos** [comment]
this "hack" is a joke guys

**Comment by Ruizlow** [comment]
Yep I bet 12 year olds in Russia make better cheats, like not even an ESP wtf?
